process/README.md

```diff
@@ -0,0 +1,249 @@
+* Remove possible legal and governance obstacles to adoption and contribution by ensuring all projects adhere to OpenSSF legal, code of conduct and IP Policy requirements.
+
+#### Gives - Review and Reporting
+
+- Copyright transfer to Linux Foundation
+- Trademark transfer to Linux Foundation
+- Add Linux Foundation as GitHub Organization Owner
+- Legal Due Diligence Documentation
+- Domain Name transfer to Linux Foundation
+- Meetup Member List
+- CLA transfer to Linux Foundation
+- Reporting required by TAC and Governing Board
+
+#### Gets - Benefits
+
+- Additional governance by OpenSSF Governing Board
+- Oversight by OpenSSF TAC
+- Access to OpenSSF Ambassador program
+- Access to OpenSSF Marketing
+- Access to Linux Foundation Legal
+- Press Releases
+- Linux Foundation Tools
+  - Insights
+  - EasyCLA
+  - Tech Support for DNS
+  - Crowdfunding
+  - Bevey
+- 1 Speaking slot at each OpenSSF events
+
+#### Sandbox Entry Requirements
+
+Working Groups and Projects being submitted to the OpenSSF at the sandbox level are intended to be the entry point for early stage projects and are not required to undergo due diligence.
+
+Sandbox Working Groups and Projects should be early-stage projects that the OpenSSF TAC believes warrant experimentation.
+
+* New Working Groups and Projects that are designed to extend one or more OpenSSF Projects with functionality or interoperability libraries.
+* Independent projects that fit the OpenSSF mission and provide potential for a novel approach to existing functional areas (or are an attempt to meet an unfulfilled need)
+* Projects commissioned or sanctioned by the OpenSSF, including initial code for OpenSSF WG collaborations, and "experimental" projects
+* Any project that realistically intends to join OpenSSF Incubation in future and wishes to lay the foundations for that
+
+#### Project Graduation Process: Sandbox to Incubating
+
+Incubating Working Groups and Projects are required to undergo due diligence as a part of the process to move from Sandbox to Incubation. Due Diligence is driven by a TAC sponsor, with two weeks for public comment before a vote is called.
+
+#### Submission Forms for Sandbox
+
+1. [Proposal](TBD)
+2. [Legal](TBD)
+
+#### Submission Forms for Sandbox to Incubating
+
+1. [Proposal](TBD)
+2. [Legal](TBD)
+
+### Incubating
+
+![Incubating](/process/incubation-process.png)
+
+#### Gives - Review and Reporting
+
+- Copyright transfer to Linux Foundation
+- Trademark transfer to Linux Foundation
+- Add Linux Foundation as GitHub Organization Owner
+- Legal Due Diligence Documentation
+- Domain Name transfer to Linux Foundation
+- Meetup Member List
+- CLA transfer to Linux Foundation
+- Reporting required by TAC and Governing Board
+- Security Audit by 3rd Party to Graduate
+
+#### Gets - Benefits
+
+- Additional governance by OpenSSF Governing Board
+- Oversight by OpenSSF TAC
+- Access to OpenSSF Ambassador program
+- Access to OpenSSF Marketing
```

Comment:
Marketing would provide Webinars, Youtube Live, Twitch, Linkedin Live, and Podcasts